### DISCUSSION TITLE
How to link ImGui with Cmake.

### DISCUSSION THREAD

**Discussion by RedFoxMain** [ocornut/imgui#7942] [New Users Build/Link/Run issues ONLY!]
Hello, everyone. I want to link imgui to my test cmake project. And i have several questions. How to do this? And is it possible to link imgui without a folder "backends"?

**Comment by nathanfourniol** [comment]
Hello, 
I'm very new to ImGui, but I tried to build imgui with Cmake this week. I ended up with this solution, https://github.com/nathanfourniol/minimal-cpp-gui. Maybe it can inspire you on how to build ImGui with Cmake.  If you have question on my cmake feel free to ask... but i'm kind of new to CMake too :)

I hope you already solve your problem by this day
